tests/test_graph_processing.py

```diff
@@ -0,0 +1,480 @@
+from contextlib import nullcontext as does_not_raise
+
+import pytest
+
+import power_system_simulation.graph_processing as gp
+
+
+def test_init_normal():
+    """
+    Normal initialization of a graph processor, should result in no errors.
+
+    1--[1]--2--[2]--3
+    ^       |
+           [3]
+            |
+            4--[4]--5
+            |
+           [5]
+            |
+            6--[6]--7--[7]--8
+    """
+
+    vertex_ids = [1, 2, 3, 4, 5, 6, 7, 8]
+    edge_ids = [1, 2, 3, 4, 5, 6, 7]
+    edge_vertex_id_pairs = [(1, 2), (2, 3), (2, 4), (4, 5), (4, 6), (6, 7), (7, 8)]
+    edge_enabled = [True, True, True, True, True, True, True]
+    source_vertex_id = 1
+
+    assert gp.GraphProcessor(vertex_ids, edge_ids, edge_vertex_id_pairs, edge_enabled, source_vertex_id)
```

Comment:
Een assert is hier niet nodig. Als er een error in je code zit, faalt deze test automatisch.
```suggestion
    gp.GraphProcessor(vertex_ids, edge_ids, edge_vertex_id_pairs, edge_enabled, source_vertex_id)
```